subgraph/src/public-lock.ts

```diff
@@ -2,7 +2,8 @@ import { Address, BigInt, log } from '@graphprotocol/graph-ts'
 
 import {
   CancelKey as CancelKeyEvent,
-  ExpirationChanged as ExpirationChangedEvent,
+  ExpirationChanged as ExpirationChangedUntilV11Event,
+  ExpirationChanged1 as ExpirationChangedEvent,
```

Comment:
This `ExpirationChanged1` is the syntax used by the subgraph autogen - so far didnt find a way to modify it